database/migrations/default/1763408675201_location_polgon_updates/up.sql

```diff
@@ -0,0 +1,416 @@
+--
+-- we have two redundant indexes that can be removed 
+-- they appear to have been created by mistake because
+-- atd_txdot_locations_pk constraint covers both of these
+--
+ALTER TABLE atd_txdot_locations 
+  DROP CONSTRAINT atd_txdot_locations_unique_id_key;
+
+DROP INDEX atd_txdot_locations_unique_id_uindex;
+
+-- convert is_intersection to boolean
+ALTER TABLE atd_txdot_locations ALTER COLUMN is_intersection DROP DEFAULT;
+
+ALTER TABLE atd_txdot_locations ALTER COLUMN is_intersection TYPE bool
+    USING CASE WHEN is_intersection = 0 THEN FALSE ELSE TRUE END;
+
+ALTER TABLE atd_txdot_locations ALTER COLUMN is_intersection SET DEFAULT FALSE;
+
+-- add new fields
+ALTER TABLE atd_txdot_locations
+    add column is_signalized boolean,
+    add column is_hin boolean,
+    add column is_service_road boolean,
+    add column council_districts integer[],
+    add column signal_eng_areas text[],
+    add column area_eng_areas text[],
+    add column zip_codes text[],
+    add column apd_sectors text[],
+    add column street_levels integer[],
```

Comment:
I love seeing these array type fields. It's the right type for the job - thanks for upgrading us here.